ts/webui/src/components/managementExp/FilterBtns.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { DefaultButton, Icon, Dropdown, DatePicker, DayOfWeek } from '@fluentui/react';
+import { EXPERIMENTSTATUS } from '../../static/const';
+import { DayPickerStrings } from './experimentConst';
+import { fillOptions } from './expFunction';
+
+interface FilterBtnsProps {
+    platform: string[];
+    selectedStatus: string;
+    selectedPlatform: string;
+    selectedStartDate: Date;
+    selectedEndDate: Date;
+    selectStatus: (_event: React.FormEvent<HTMLDivElement>, item: any) => void;
+    selectPlatform: (_event: React.FormEvent<HTMLDivElement>, item: any) => void;
+    getSelectedData: (type: string, date: Date | null | undefined) => void;
+    setSearchSource: () => void;
+}
+
+class FilterBtns extends React.Component<FilterBtnsProps, {}> {
+    constructor(props: FilterBtnsProps) {
+        super(props);
+    }
+
+    render(): React.ReactNode {
+        const {
+            platform,
+            selectedStatus,
+            selectedPlatform,
+            selectedStartDate,
+            selectedEndDate,
+            selectStatus,
+            selectPlatform,
+            getSelectedData,
+            setSearchSource
+        } = this.props;
+
+        return (
+            <React.Fragment>
+                <Dropdown
+                    label='Status'
+                    selectedKey={selectedStatus}
+                    onChange={selectStatus.bind(this)}
+                    placeholder='Select an option'
+                    options={fillOptions(EXPERIMENTSTATUS)}
+                    className='filter-condition-status'
+                />
+                <Dropdown
```

Comment:
When there is only one platform, this dropdown does not work.

Reply:
Thanks for finding that! I'll fix this issue.